cmd/automountServiceAccountToken_fixes_test.go

```diff
@@ -6,10 +6,52 @@ func TestFixServiceAccountTokenDeprecatedV1(t *testing.T) {
 	assert, resource := FixTestSetup(t, "service_account_token_deprecated_v1.yml", auditAutomountServiceAccountToken)
 	switch typ := resource.(type) {
 	case *ReplicationControllerV1:
+		assert.Equal("fakeDeprecatedServiceAccount", typ.Spec.Template.Spec.ServiceAccountName)
 		assert.Equal("", typ.Spec.Template.Spec.DeprecatedServiceAccount)
 	}
 }
 
+func TestFixServiceAccountTokenDeprecatedV2(t *testing.T) {
+	assert, resources := FixTestSetupMultipleResources(t, "service_account_token_deprecated_multiple_resources_v1.yml", auditAutomountServiceAccountToken)
+	for index := range resources {
+		switch t := resources[index].(type) {
+		case *CronJobV1Beta1:
+			assert.Equal("fakeDeprecatedServiceAccount", t.Spec.JobTemplate.Spec.Template.Spec.ServiceAccountName)
+			assert.Equal("", t.Spec.JobTemplate.Spec.Template.Spec.DeprecatedServiceAccount)
+		case *DaemonSetV1:
```

Comment:
I think you are missing a `DaemonSet` for `extensions/v1beta1` here.

Reply:
*DaemonSetV1Beta1 is the DaemonSet for extensions/v1beta 

`// DaemonSetV1Beta1 is a type alias for the v1beta1 version of the k8s extensions API.
type DaemonSetV1Beta1 = extensionsv1beta1.DaemonSet`

Reply:
appsv1/beta for DaemonSet is not a valid resource, that's why it was getting removed. I got rid of it now